<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AERPScriptEditDlg</class>
 <widget class="QDialog" name="AERPScriptEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="DBCodeEdit" name="txtScript" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbSave">
     <property name="text">
      <string>Guardar</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/aplicacion/images/ok.png</normaloff>:/aplicacion/images/ok.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbCancel">
     <property name="text">
      <string>Salir</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/generales/images/close.png</normaloff>:/generales/images/close.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DBCodeEdit</class>
   <extends>QWidget</extends>
   <header>widgets/dbcodeedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
